shared/teams/team/settings-tab/retention/index.js

```diff
@@ -196,22 +196,22 @@ class _RetentionPicker extends React.Component<Kb.PropsWithOverlay<Props>, State
           </Kb.Box2>
           <Kb.Icon type="iconfont-caret-down" inheritColor={true} fontSize={7} />
         </Kb.ClickableBox>
-        {this.props.showOverrideNotice && (
-          <Kb.Text style={{marginTop: Styles.globalMargins.xtiny}} type="BodySmall">
-            Individual channels can override this.
-          </Kb.Text>
-        )}
         {this.props.policyIsExploding && (
           <Kb.Box2 direction="horizontal" alignItems="center" fullWidth={true} gap="xtiny">
             <Kb.Text type="BodySmall">Participants will see their message explode.</Kb.Text>
-            <Kb.Icon
-              color={Styles.globalColors.black_75}
-              fontSize={Styles.isMobile ? 44 : 35}
-              style={boomIconStyle}
-              type="iconfont-boom"
-            />
+            <Kb.Box style={boomIconBoxStyle}>
```

Comment:
This goes off the right side of the screen in the infopanel on on the iPhone 5s sim. Weirdly, the text wraps correctly on the team settings page.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/11968340/52370039-f5574200-2a1f-11e9-96a9-f59437ef9e33.png">
<img width="459" alt="image" src="https://user-images.githubusercontent.com/11968340/52370058-fe481380-2a1f-11e9-82ce-c67bbff4e125.png">

Reply:
@buoyad Removing a `width: '100%'` got the wrap working, what do you think?